sympy/geometry/ellipse.py

```diff
@@ -1267,6 +1267,21 @@ def vradius(self):
         """
         return self.args[2]
 
+    @property
+    def centroid(self):
+        """Returns the centroid of ellipse.
+
+        Returns
+        =======
+
+        centroid : Point
+
+        Examples
+        ========
+
+        """
+        return self.center
```

Comment:
These seems redundant. Why is a `centroid` method needed? The centroid is only different if the density is not uniform.

Reply:
I implemented it just for the uniformity. Polygon and parabola have the `centroid` method.
should I remove it?

Reply:
We should be consistent and only use center or centroid. Centroid sounds like the better name of the method to me. Disregard my comment about density, was thinking about non-pure geometric objects. I do not recommend adding this now, but you can propose a solution to fix the consistency issue. A PR that has deprecations of `center()` and introductions of `centroid()` could work. But if the `center()` method is in wide use, maybe we should stick with it. Whatever the case, I don't think it is a good idea to continue adding redundant methods.